packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/file_storage/file_storage.web.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:convert';
+import 'dart:html';
+
+/// Save constraint locally to web storage
+Future<void> saveConstraintLocally(Map<String, dynamic> constraint) async {
+  window.localStorage['logging_constraint'] = jsonEncode(constraint);
```

Comment:
Namespace key
Save key in const global private in this file

```suggestion
  window.localStorage['aws.cloudwatch.logging_constraints'] = jsonEncode(constraint);
```